robotidy/cli.py

```diff
@@ -266,6 +266,9 @@ def print_transformers_list(target_version: int):
         " excluded ones. (Useful if you simply want to add to the default)"
     ),
 )
+@click.option(
+    "--ignore-gitignore", is_flag=True, show_default=True, help="Do not skip files listed in **.gitignore** files."
```

Comment:
```suggestion
    "--ignore-gitignore", is_flag=True, show_default=True, help="Do not skip files listed in **.gitignore** file."
```

Reply:
Files - there could be several ``gitignores`` files and robotidy *smartly* recognize them:

```
.gitignore -> skip test.robot
test.robot
test2.robot
nested/
  .gitignore -> skip test2.robot
  test.robot
  test2.robot
```
Above will yield only test2.robot from the main directory.